app/helpers/search_helper.rb

```diff
@@ -85,6 +85,20 @@ def ordering
     end
   end
 
+  # Transforms controller params from get request into query args for a search
+  # @return [Hash]
+  def get_query_args
```

Comment:
Why not just make the export button a POST? Bear in mind that we can add an unbound set of filters in the search UI. I'm concerned that a sufficiently complicated search will result in a query string that will be longer than the general URL limit of 2000 characters.

I agree that the current format of agg_filters in particular isn't perfect, but some of that just comes from backwards compatibility with a prior implementation. I'm certainly open to improvements there long term.